src/ExtensionsSample/Program.cs

```diff
@@ -66,7 +69,8 @@ public static void Main(string[] args)
                 typeof(SendGridSamples),
                 typeof(TableSamples),
                 typeof(TimerSamples),
-                typeof(WebHookSamples));
+                typeof(WebHookSamples),
+                typeof(ApiHubSamples));
```

Comment:
There is no need to configure anything anymore to have ApiHub stuff work, the samples are now much easier and work with local file system. they can be run without ANY extra configuration than other regular stuff.